app/packs/src/utilities/MathUtils.js

```diff
@@ -56,9 +56,42 @@ const formatBytes = (bytes, decimals = 2) => {
   return `${parseFloat((bytes / (k ** i)).toFixed(dm))} ${sizes[i]}`;
 };
 
+function parseNumericString(numberString) {
```

Comment:
As this a potential useful function in many contexts, It would be nice to see how this function behave without reading the tests or looking into the implementation. fo example with some JSDoc documentation?

Reply:
Good point, see 59e2d74.

Reply:
Sorry for being annoying. Isn't the doc-string usually above the function signature?

Reply:
Good catch, I didn't know the convention. Fixed with 0665a60.